src/coverlet.core/Coverage.cs

```diff
@@ -228,7 +232,7 @@ private void CalculateCoverage()
         {
             foreach (var result in _results)
             {
-                if (!File.Exists(result.HitsFilePath))
+                if (!_fileSystem.Exists(result.HitsFilePath))
```

Comment:
missing code below at line 256 `using (var fs = new FileStream(result.HitsFilePath, FileMode.Open))` do a search for FileStream in this file.

Reply:
Done